packages/expo-sqlite/localStorage/install.js

```diff
@@ -0,0 +1,4 @@
+if (process.env.EXPO_OS !== 'web') {
+  const { installGlobal } = require('../build/Storage');
+  installGlobal();
+}
```

Comment:
Just to confirm: on web, will the bundler remove 100% of this code by statically evaluating `if (process.env.EXPO_OS !== 'web')`?